Dokan/Dokan/Scenes/ReviewProduct/ReviewProductViewController.swift

```diff
@@ -0,0 +1,196 @@
+                self.configureNavBar(rating: rating)
+            }
+        }
+
+        viewModel.updateLoadingStatus = { [weak self] () in
+            guard let self = self else { return }
+
+            switch self.viewModel.state {
+            case .empty, .error:
+                self.mainStackView.isHidden = true
+            case .loading:
+                self.mainStackView.isHidden = false
+                self.loadingCell = true
+                self.mainStackView.startSkeletonView()
+            case .loaded:
+                self.mainStackView.isHidden = false
+                self.loadingCell = false
+                self.mainStackView.stopSkeletonView()
+            }
+        }
+
+        viewModel.reloadTableViewClosure = { () in
+            DispatchQueue.main.async { [weak self] in
+                guard let self = self else { return }
+
+                self.reviewCount = self.viewModel.numberOfCells
+                self.reviewProductTableView.reloadData()
+            }
+        }
+
+        viewModel.configureFetchRatingDetails { [weak self] reviews in
+            guard let self = self else { return }
+            guard let reviews = reviews else {
+                return
+            }
+            self.ratingDetailsView.ratingDetails = reviews
+        }
+
+        viewModel.loadReviews()
+    }
+}
+
+// MARK: - Private Handlers
+
+private extension ReviewProductViewController {
+
+    // MARK: Show error alert
+
+    func showErrorAlert(error: String) {
+        UIAlertController.Builder()
+            .title(Constants.errorAlertTitle)
+            .message(error)
+            .addActionWithTitle(Constants.errorAlertBackButton, style: .default, handler: { _ in
+                // TODO: back to previous screen
+            })
+            .addActionWithTitle(Constants.errorAlertTryAgainButton, style: .default, handler: { _ in
+                self.viewModel.loadReviews()
+            })
+            .show(in: self)
+    }
+}
+
+// MARK: - TableView
+
+extension ReviewProductViewController: UITableViewDelegate, UITableViewDataSource {
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return reviewCount
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: Constants.cellReuseIdentifier, for: indexPath) as! ReviewerTableViewCell
+
+        if loadingCell {
+            cell.startSkeletonView()
+        } else {
+            cell.stopSkeletonView()
+            let cellVM = viewModel.getCellViewModel(at: indexPath)
+            cell.configureCell(viewModel: cellVM)
+        }
+        return cell
+    }
+}
+
+// MARK: - Constants
+
+extension ReviewProductViewController {
+
+    private enum Constants {
+        static let tableViewCellName = "ReviewerTableViewCell"
+        static let cellReuseIdentifier = "ReviewerTableViewCell"
```

Comment:
We've helpers for register and dequeue so there's no reason for hardcoded values.